source/package.json

```diff
@@ -25,6 +25,7 @@
   "private": true,
   "dependencies": {
     "@digipolis-gent/modal": "^1.0.3",
+    "fs-extra": "^10.0.0",
```

Comment:
This belongs in https://github.com/StadGent/fractal_styleguide_gent-base/blob/5.x-dev/package.json#L11